inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/DataNodeEntity.java

```diff
@@ -31,6 +31,7 @@ public class DataNodeEntity implements Serializable {
     private static final long serialVersionUID = 1L;
     private Integer id;
     private String name;
+    private String dataNodeDisplayName;
```

Comment:
No need to name dataNodeDisplayName, just call it **dispalyName**

Reply:
done